web/packages/teleterm/package.json

```diff
@@ -18,7 +18,7 @@
     "build-renderer": "webpack build --config webpack.renderer.prod.config.js --progress",
     "build-native-deps": "electron-builder install-app-deps",
     "package": "electron-builder build --config electron-builder-config.js --publish never -c.extraMetadata.name=teleport-connect",
-    "generate-grpc-shared": "npx -y --target_arch=x64 --package=grpc_tools_node_protoc_ts@5.3.2 --package=grpc-tools@1.11.2 -- grpc_tools_node_protoc -I=src/sharedProcess/api/proto --ts_out=service=grpc-node,mode=grpc-js:src/sharedProcess/api/protogen --grpc_out=grpc_js:src/sharedProcess/api/protogen --js_out=import_style=commonjs,binary:src/sharedProcess/api/protogen src/sharedProcess/api/proto/*.proto"
+    "generate-grpc-shared": "npx -y --target_arch=x64 --package=grpc_tools_node_protoc_ts@5.3.2 --package=grpc-tools@1.12.4 -- grpc_tools_node_protoc -I=src/sharedProcess/api/proto --ts_out=service=grpc-node,mode=grpc-js:src/sharedProcess/api/protogen --grpc_out=grpc_js:src/sharedProcess/api/protogen --js_out=import_style=commonjs,binary:src/sharedProcess/api/protogen src/sharedProcess/api/proto/*.proto"
```

Comment:
This is related to internal protobufs used by Connect. It looks gnarly but we didn't even need to run this command since we added [ptyHostService.proto](https://github.com/gravitational/teleport/blob/79ebdc89c8750b31c5cd7552b1845e61eee88994/web/packages/teleterm/src/sharedProcess/api/proto/ptyHostService.proto).

Next time we do anything with those protos, we should move them to proto as thanks to the webapps merge and #20810 we now have one convenient location for all protos. Actually, I'll add a comment to ptyHostService.proto about it.